lms/templates/footer.html

```diff
@@ -1,7 +1,7 @@
 ## mako
```

Comment:
@jcdyer, this change is already present in hawthorn. So it can be removed from the theme.